app/src/lib/ssh/ssh-key-passphrase.ts

```diff
@@ -0,0 +1,78 @@
+import { getFileHash } from '../file-system'
+import { TokenStore } from '../stores'
+
+const appName = __DEV__ ? 'GitHub Desktop Dev' : 'GitHub Desktop'
+const SSHKeyPassphraseTokenStoreKey = `${appName} - SSH key passphrases`
+
+async function getHashForSSHKey(keyPath: string) {
+  return getFileHash(keyPath, 'sha256')
+}
+
+/** Retrieves the passphrase for the SSH key in the given path. */
+export async function getSSHKeyPassphrase(keyPath: string) {
+  try {
+    const fileHash = await getHashForSSHKey(keyPath)
+    return TokenStore.getItem(SSHKeyPassphraseTokenStoreKey, fileHash)
+  } catch (e) {
+    log.error('Could not retrieve passphrase for SSH key:', e)
+    return null
+  }
+}
```

Comment:
These are the same functions that previously lived in `ssh.ts`, just brought them here to live with other passphrase-related functions that I needed to add.